### PR TITLE
#47 change method registration

### PR DESCRIPTION
**Pull request by aliquispassim**
- changed Register in AccountController
- changed RegisterUser in ServiceAccount
- changed the return type of IServiceAccount.RegisterUser

### CODE REVIEW

**Review by pinkevmladchy — APPROVED** [review]
good work

**Review by LizaShemetovska — APPROVED** [review]

File: AccountService.Domain/Services/ServiceAccount.cs [hunk 1]
```diff
@@ -42,15 +41,17 @@ public async Task RegisterUser(RegisterApiModel userRequest)
                 if (user.UserData.ServiceNumber == null)
                 {
                     await _userManager.AddToRoleAsync(user, "participant");
+                    return new ResponseApiModel<HttpStatusCode>(HttpStatusCode.OK, true, "You have succesfully registered");
```
Comment (SashaKorniichuk) [inline AccountService.Domain/Services/ServiceAccount.cs]: You can get message for RestException from Resources.resx

File: AccountService.Domain/Services/ServiceAccount.cs [hunk 2]
```diff
@@ -42,15 +41,17 @@ public async Task RegisterUser(RegisterApiModel userRequest)
...
                 }
                 else
                 {
                     await _userManager.AddToRoleAsync(user, "inspector");
+                    return new ResponseApiModel<HttpStatusCode>(HttpStatusCode.OK, true, "You have successfully registered as an inspector");
```
Comment (SashaKorniichuk) [inline AccountService.Domain/Services/ServiceAccount.cs]: You can get message for RestException from Resources.resx

**Review by SashaKorniichuk — CHANGES_REQUESTED** [review]

File: AccountService.Domain/Services/ServiceAccount.cs (hunk 1 above)
Comment (aliquispassim) [reply AccountService.Domain/Services/ServiceAccount.cs]: Done

File: AccountService.Domain/Services/ServiceAccount.cs [hunk 3]
```diff
@@ -42,15 +41,17 @@ public async Task RegisterUser(RegisterApiModel userRequest)
                 if (user.UserData.ServiceNumber == null)
```
Comment (LizaShemetovska) [inline AccountService.Domain/Services/ServiceAccount.cs]: you can delete this check because only the participant can register

Comment (aliquispassim) [reply AccountService.Domain/Services/ServiceAccount.cs]: alright, done

**Review by SashaKorniichuk — APPROVED** [review]

**Pull request merged**